core/specfem/chunk_element/impl/field.hpp

```diff
@@ -285,6 +289,17 @@ class field : public specfem::data_access::Accessor<
     return m_data(indices...);
   }
 
+  KOKKOS_FORCEINLINE_FUNCTION auto &
+  operator()(const specfem::point::index<dimension_tag, using_simd> &index) {
+    return m_data(index);
+  }
+
+  KOKKOS_FORCEINLINE_FUNCTION typename value_type::value_type &
+  operator()(const specfem::point::index<dimension_tag, using_simd> &index,
+             const int &icomp) {
+    return m_data(index, icomp);
+  }
```

Comment:
I think it would be possible to pass this mapped index too. 
```suggestion
  template <typename IndexType, typename std::enable_if_t<specfem::data_access::is_index_type<IndexType>::value && specfem::data_access::is_point_type<IndexType>::value, int= 0>>
  KOKKOS_FORCEINLINE_FUNCTION auto &
  operator()(const IndexType &index) {
    return m_data(index);
  }

  template <typename IndexType, typename std::enable_if_t<specfem::data_access::is_index_type<IndexType>::value && specfem::data_access::is_point_type<IndexType>::value, int= 0>>
  KOKKOS_FORCEINLINE_FUNCTION typename value_type::value_type &
  operator()(const IndexType &index,
             const int &icomp) {
    return m_data(index, icomp);
  }
```